packages/client/src/components/SafeSettings/components/Assets/AddVault.js

```diff
@@ -15,16 +17,19 @@ const AddVault = ({ onCancel, onNext }) => {
     displayText: type[1].displayName,
   }));
 
-  const [coinType, setCoinType] = useState(coinTypes[0].itemValue);
+  const [coinType, setCoinType] = useState("Select Token");
```

Comment:
Suggest 
1. renaming the state from coinType to selectedCoinType
2. initiating with a nullish value, `""` or `undefined`, 
3. prepend coinTypes with {itemValue:undefined, displayText:""}, 
4. have dropdown component accepting a defaultText property for displaying "Select Token" 
why:
"Select Token" is not one of the coin types, on validating the input, it's easier to check against a nullish value than a string

Reply:
Renamed the state, set it to undefined and added a defaultText for Dropdown component. I didn't add a new coin type with undefined value to coin types because that would make it possible to reselect and would appear in a dropdown. Please take a look and tell me what do you think about the new approach :)

Reply:
Love the new approach!